WEB-INF/classes/uon/seng2050/assignment/model/Article.java

```diff
@@ -11,7 +12,6 @@
   private String answer;
```

Comment:
This should be a foreign key (UUID). Also just add category and subcategory please :)